app/src/main/kotlin/org/fossify/filemanager/fragments/MyViewPagerFragment.kt

```diff
@@ -88,6 +90,13 @@ abstract class MyViewPagerFragment<BINDING : MyViewPagerFragment.InnerBinding>(c
         }
     }
 
+    protected fun String.normalizeText(): String {
+        val normalizer = Normalizer2.getNFDInstance()
+        return normalizer.normalize(this)
+            .replace("\\p{M}".toRegex(), "")
+            .lowercase(Locale.getDefault())
+    }
+
```

Comment:
There's a `String.normalizeString()` extension function for this in the commons lib, please use that.
```suggestion
```